### PR TITLE
Bookstore setup

### PR DESCRIPTION
**Pull request by summyalena**
### I created the setup for the bookstore react app. I ensured that I followed the requirements for the setup.

- [ ] Initialise React app.
- [ ] The building blocks of your app should be set as re-usable components. Create a directory for your components.
- [ ] Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
- [ ] Books - the default view:
- [ ] Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet).
- [ ] Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- [ ] Should have a form for adding a book (no functionality yet).
- [ ] Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
- [ ] Categories:
- [ ] Should display a button "Check status" only.
- [ ] Styling is not required at this point.

### CODE REVIEW

File: src/category.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
```
Comment (Deepakdanger) [inline src/category.js]: - [ ] Kindly remove this line & solve the linter manually.

File: src/category.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import React from 'react'
+
+function category() {
+  return (
+    <div>
+      Category
+    </div>
```
Comment (Deepakdanger) [inline src/category.js]: - [ ] It should display the button with "check status"

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @summyalena , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:
success

## Hey @summyalena :wave:

Great work on implementing the requested changes by the previous reviewer. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Following the GitFlow
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following the Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**